### PR TITLE
Added `pl_to_gp` (phred-scaled -> linear genotypes)

### PR DESCRIPTION
**Pull request by lfrancioli**

### CODE REVIEW

**Comment by tpoterba** [comment]
I'd be okay with adding this, but only if it's implemented in Python

**Comment by lfrancioli** [comment]
I wanted to take advantage of the pre-computed phred-scale -> linear table in the scala. Admittedly it can be done in python too, but a little more cumbersome.

**Comment by tpoterba** [comment]
you can still do this in Python with `literal`!

**Comment by lfrancioli** [comment]
OK, I'll suggest a python implementation.

File: python/hail/expr/functions.py [hunk 1]
```diff
@@ -853,6 +853,33 @@ def pl_dosage(pl) -> Float64Expression:
...
+        Array of Phred-scaled genotype likelihoods
+
+    Returns
+    -------
+   :class:`.ArrayNumericExpression` of type :py:data:`.tfloat64`
+    """
+    max_phred_in_table = 8192
+    phred_table = hl.literal([10 ** (-x/10.0) for x in builtins.range(max_phred_in_table)])
+    gp = hl.bind(lambda pls: pls.map(lambda x: hl.cond(x >= max_phred_in_table, 10 ** (-x/10.0), phred_table[x])), pl)
+    return gp / hl.sum(gp)
```
Comment (tpoterba) [inline python/hail/expr/functions.py]: might want another `bind` here for now :(

File: python/hail/expr/functions.py [hunk 2]
```diff
@@ -853,6 +853,33 @@ def pl_dosage(pl) -> Float64Expression:
...
+    --------
+    .. doctest::
+
+        >>> hl.eval_expr(hl.pl_to_gp([0, 10, 100]))
+        [0.9090909090082644, 0.09090909090082644, 9.090909090082645e-11]
+
+    Parameters
+    ----------
+    pl : :class:`.ArrayNumericExpression` of type :py:data:`.tint32`
+        Array of Phred-scaled genotype likelihoods
```
Comment (tpoterba) [inline python/hail/expr/functions.py]: period at the end

**Pull request closed without merging**